### ISSUE TITLE
Image field mapped to wrong Carbon Black field?

### ISSUE THREAD

**Issue by hegga** [Neo23x0/sigma#814]
https://github.com/Neo23x0/sigma/blob/d97d2ced8274bd2972b5d10ff8a56fc6c2c0242f/tools/config/carbon-black.yml#L16

The Image field in Sysmon logs contains the full path of the executable for the process[1][2], but is mapped to the process_name field in Carbon Black which only contains the filename[3]. So should not the Image field be mapped to the path field in Carbon Black instead? As the path field contains the full path of the executable.

```
# this will give no results in Carbon Black
process_name:C:\Windows\System32\sethc.exe

# this will work, but not limit the executable to be located in C:\Windows\System32
process_name:sethc.exe

# this will work
path:C:\Windows\System32\sethc.exe

# this will also work, but not limit the executable to be located in C:\Windows\System32
path:sethc.exe
```

[1] - https://docs.microsoft.com/en-us/media/landing/sysinternals/event-screen-optimized.png
[2] - https://github.com/Neo23x0/sigma/wiki/Taxonomy
[3] - https://developer.carbonblack.com/resources/query_overview.pdf

**Comment by ozirus** [comment]
I think the problem is rules if you talk about the ones in process_creation rule folder. For example in win_wmi_persistence_script_event_consumer.yml rule, Image: C:\WINDOWS\system32\wbem\scrcons.exe line should be written as *.\scrcons.exe like others. Other version should be in sysmon rule folder (if they also use Sysmon specific props) which win_wmi_persistence_script_event_consumer doesn't.

**Comment by hegga** [comment]
Closed by #982
